sdk/src/types/block/rand/payload.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+use super::signature::rand_signature;
```

Comment:
I'm annoying but there is already a rand import, I'd just put it with the others